agent/src/bpf/audit.h

```diff
@@ -30,10 +30,15 @@ struct audit_event_header_st
   unsigned long int ktime;	/* u64 */
 };
 
+#ifndef BPF_BUILD_ID_SIZE
+#define BPF_BUILD_ID_SIZE 20
```

Comment:
In which situations would this not already be defined in `linux/bpf.h`?

Reply:
We use `vmlinux.h` generated with `bpftool btf dump file /sys/kernel/btf/vmlinux format c` instead of `linux/bpf.h`, which doesn't include macro definitions.

Reply:
I think we could rely on BPF [CO-RE](https://nakryiko.com/posts/bpf-portability-and-co-re/) for that, as in the code I've just updated.